gruel/src/main/groovy/com/glasstowerstudios/gruel/GruelPlugin.groovy

```diff
@@ -16,6 +17,7 @@ public class GruelPlugin implements Plugin<Project> {
     gruelExtension.setProject(aProject);
 
     aProject.extensions.create("hipchat", HipChatExtension);
+    def jiraExtension = aProject.extensions.create("jira", JiraExtension);
```

Comment:
Yeah, I'm actually going to make it part of the Gruel extension in a future modification.